pkg/labelsfilter/filter_test.go

```diff
@@ -163,3 +165,50 @@ func (s *LabelsPrefCfgSuite) TestFilterLabelsDocExample(c *C) {
 	c.Assert(len(filtered), Equals, 6)
 	c.Assert(filtered, checker.DeepEquals, wanted)
 }
+
+func TestFilterLabelsByRegex(t *testing.T) {
+	type args struct {
+		excludePatterns []*regexp.Regexp
+		labels          map[string]string
+	}
+	tests := []struct {
```

Comment:
Nit. Could make sense to additionally cover the case of multiple filters.

Reply:
ACK, will add one more test.

Reply:
Done